packages/contracts/contracts/LenderCommitmentForwarder.sol

```diff
@@ -192,25 +318,73 @@ contract LenderCommitmentForwarder is TellerV2MarketForwarder {
             "Commitment has expired"
         );
 
-        CreateLoanArgs memory createLoanArgs;
-        createLoanArgs.marketId = _marketId;
-        createLoanArgs.lendingToken = _tokenAddress;
-        createLoanArgs.principal = _principal;
-        createLoanArgs.duration = _loanDuration;
-        createLoanArgs.interestRate = _interestRate;
+        require(
```

Comment:
Wouldn't this fail if the collateral is USDC & ETH is Loan Amount. For Ex: 2000 USDC as collateral for 1 ETH loan Volume ?